### DISCUSSION TITLE
Configure Onboarding PR

### DISCUSSION THREAD

**Discussion by kevin-matthews-vc** [renovatebot/renovate#33338] [Suggest an Idea]
### How are you running Renovate?

A Mend.io-hosted app

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

I would like to modify the onboarding config within renovate, i know this can be done on self hosted, but it seems such an easy solution to add to the mend setup and i cannot see how to do it.

Just little things like being able to set the extend rule to include what i want rather than having to modify every PR that gets created, or adding things such as `ignoreTests` so the renovate configure PR can be merged without having to circumvent security.

Am I missing an option or is this available elsewhere?

Thanks

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
It is a feature request. I have changed the category.

**Comment by rarkins** [comment]
Use "inheritConfig" for full customization of onboardingConfig. Alternatively you can add a renovate-config repo in your org with a default.json and it will automatically be used as the onboarding config "extends" value for onboarding

**Reply by kevin-matthews-vc** [reply]
How do you use inherit config? Already have a internal renovate config repo which has a different name

**Reply by rarkins** [reply]
It's described in the docs. Please read that and ask about any specifics

**Reply by kevin-matthews-vc** [reply]
> If you use a Mend-hosted app, you can not change the values for the inheritConfigFileName and the inheritConfigRepoName config options.

This seems to me that unless I change my base renovate repository name I won't be able to use it.

Can there not be be an option to modify the inherit repo and inherit file name in the UI?

Thanks

**Reply by rarkins** [reply]
There could be, but it's not planned

**Reply by kevin-matthews-vc** [reply]
ok, ill just have to manually change the PRs for now then, or plan to rename the base repository. Thx
